runtime/frequency/src/weights/orml_vesting.rs

```diff
@@ -47,7 +49,7 @@ impl<T: frame_system::Config> orml_vesting::WeightInfo for SubstrateWeight<T> {
 	// Storage: System Account (r:2 w:2)
 	// Storage: Balances Locks (r:1 w:1)
 	fn vested_transfer() -> Weight {
```

Comment:
These came out fairly different from https://github.com/AcalaNetwork/Acala/blob/master/runtime/acala/src/weights/orml_vesting.rs which in theory uses the same reference hardware.

Reply:
Hmmm... Acala does use different parameters for steps and repeat: https://github.com/AcalaNetwork/Acala/blob/7eaccb4163619c5462f99974280d30341297d1cd/Makefile#L250-L251

Reply:
50 steps would be more accurate but also slower.

Reply:
update:  I ran the weights with 50.